lib/model/model_test.go

```diff
@@ -952,6 +952,9 @@ func TestIgnores(t *testing.T) {
 	m.ServeBackground()
 	defer m.Stop()
 
+	// Let initial setup operations (scan, ...) happen
+	time.Sleep(time.Duration(10) * time.Millisecond)
```

Comment:
Very suspicious here. What are we waiting for exactly? If it's a scan, we should just do a scan instead as the 10 ms will probably not be enough on some random computer out there.